pkg/R/sparkR.R

```diff
@@ -36,14 +36,17 @@ sparkR.stop <- function(env) {
   if (exists(".sparkRjsc", envir = env)) {
     sc <- get(".sparkRjsc", envir = env)
     callJMethod(sc, "stop")
+    rm(".sparkRjsc", envir = env)
   }
 
```

Comment:
I have another question. Do ".sparkRjsc" and ".sparkRCon" always exists simultaneously or is it possible for one to exist and not the other?
If so, these two if statements can be merged together into one if statement like, 
if (exists(".sparkRjsc", envir = env)&&exists(".sparkRCon", envir = env)) {...}